types/address.go

```diff
@@ -20,6 +20,13 @@ const (
 	// Bech32PrefixAccAddr defines the Bech32 prefix of an account's address
 	Bech32MainPrefix = "cosmos"
```

Comment:
If coinType changes, I think the bech32 prefixes would also need to change.

Reply:
Yes, `prefix` #2614  and `coinType`#4300 should be matched.
If this PR was merged, they are matched. :)

```go
func main() {
	cdc := app.MakeCodec()

	config := sdk.GetConfig()
	config.SetBech32PrefixForAccount(sdk.Bech32PrefixAccAddr, sdk.Bech32PrefixAccPub)
	config.SetBech32PrefixForValidator(sdk.Bech32PrefixValAddr, sdk.Bech32PrefixValPub)
	config.SetBech32PrefixForConsensusNode(sdk.Bech32PrefixConsAddr, sdk.Bech32PrefixConsPub)
	config.SetCoinType(sdk.CoinType)
	config.SetFullFundraiserPath(sdk.FullFundraiserPath)
	config.Seal()
        ......
}
```

![image](https://user-images.githubusercontent.com/7357939/57352277-7a750300-7197-11e9-876a-6c442f1778c3.png)